devel/build

```diff
@@ -36,14 +36,14 @@ if ! docker buildx inspect "$builder" &>/dev/null; then
     # However, if this is changed and it was previously run on your machine,
     # you may need to remove the builder manually before running the script:
     #     docker builder rm "nextstrain-builder"
-    docker buildx create --name "$builder" --driver docker-container
+    docker buildx create --name $builder --driver docker-container
```

Comment:
This is an odd change to be making to me, as quoting variable substitutions is a good defensive practice against future meta-chars in the variable value. I'd expect to be changing the `devel/*` programs to _add_ quotes to variable substitutions!

(There's also still a `"$builder"` form a few lines above in this file, so not sure why that wouldn't be changed here too…)

Reply:
Good point, updated to use quotes: 257851bb4de28b899783b0b2dbe8d6c496a82a52

Reply:
Ok… quotes around all strings feels unnecessary, but it's also fine.

Reply:
I'd also be inclined to make such style changes (esp. those "for consistency" :-) across all files at once instead of just one.

Reply:
Toned back the quoting and made changes in other files: aa28baf42fb3e0255ab5f2ecad6823e2864cb628